test/javascripts/components/chat-channel-preview-card-test.js

```diff
@@ -86,5 +86,35 @@ discourseModule(
         "it shows a link to browse all channels"
       );
     });
+
+    test("closed channel", async function (assert) {
```

Comment:
I don't think it is the concern of this component test to know about the different state of the different channels. I think we only need a single test here where `channel.isOpen` is `false`.

Reply:
`isOpen` is derived from the channel status and cannot be set directly:

https://github.com/discourse/discourse-chat/blob/31789e24c1eeda0b992b5417f07802b52bf42957/assets/javascripts/discourse/models/chat-channel.js#L79-L82

But I can just remove everything except the `closed channel` test here